admin/app/components/solidus_admin/sidebar/item/component.rb

```diff
@@ -49,9 +61,13 @@ def nested_items
     return unless @item.children?
 
     tag.nav(
-      class: "hidden group-hover:block"
+      class: "hidden group-hover:block group-[.active]:block"
```

Comment:
I'm trying to understand if this can be replaced with ` class: "#{:hidden unless active?} group-hover:block"`, since `#active?` is fully static

Reply:
Good call! Fixed.